modyn/evaluator/internal/metrics/abstract_evaluation_metric.py

```diff
@@ -52,6 +52,13 @@ def get_name(self) -> str:
         """
         raise NotImplementedError()
 
+    @abstractmethod
```

Comment:
I find this a bit odd, I'd rather create separate Metric objects if they use independent states.

Reply:
Yes, I'd also say we should instantiate new objects

Reply:
Thanks. Will do.

Reply:
done